eks_fargate/tests/test_eks_fargate.py

```diff
@@ -0,0 +1,25 @@
+# (C) Datadog, Inc. 2020
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import os
+
+from datadog_checks.eks_fargate import EksFargateCheck
+
+
+def test_eksfargate(aggregator):
+    os.environ["DD_KUBERNETES_KUBELET_NODENAME"] = "fargate-foo"
+    os.environ["HOSTNAME"] = "bar"
```

Comment:
```
with EnvVars({ ... }):
```

Reply:
Is this just cosmetic or my approach is incorrect ?

Reply:
`with EnvVars` restores the values of env vars after the end of the block.